### ISSUE TITLE
Implement project status badge

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#7618]
As a Moped user, I want to see a concise status label so that I can quickly know a project's status without sifting through other project summary fields.

![project_status](https://user-images.githubusercontent.com/34947604/140184001-1522b687-9f31-43ab-8eeb-0e0b8dad757f.png)

## Badges
<img width="1212" alt="Screen Shot 2021-11-24 at 3 20 19 PM" src="https://user-images.githubusercontent.com/1463708/143314359-b3317bdb-f6a7-45c7-887a-a46dd43c72b8.png">

[XD](https://xd.adobe.com/view/74e0ba56-e00f-49d8-9a35-7ec2cf144554-21b3/screen/40a77190-463a-49e2-8f72-c6135ad45052/)

Use [chip](https://v4.mui.com/components/chips/) component, but with
- Extra padding + corner radius as needed to match comps
- font-weight: medium (500)
- font-size: 16pt (21px)

### Colors 
_Color - Hexcode - MUI Description_
Red - `#F44336` - Error Main
- Canceled

Grey - `#E0E0E0` - Default 300
- On hold

Yellow - `#FFB74D` - Warning Light
- Potential

Green - `#4CAF50` - Success Main
- Planned
- Preliminary engineering
- Scoping
- Preliminary design
- Design
- Pre-construction
- Bid/Award/Execution
- Construction-ready
- Construction
- Post-construction

Blue - `#1976D2` - Info Dark
- Complete

### Icons
- [Circle Cancel](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Acancel%3A)
- [Circle Pause](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Apause_circle%3A)
- [Circle Dash](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aremove_circle_outline%3A)
- [Circle Play](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aplay_circle%3A)
- [Circle Check](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Acheck_circle%3A)

## Data Model

**PLEASE NOTE THAT THE `moped_project_status` table should just be `moped_status`**

<img width="1147" alt="status and phase erd #2" src="https://user-images.githubusercontent.com/34947604/141185327-d953e6b2-60df-4c6d-9f12-47ef32bd971e.png">

Notes:

- **Please note that `current_phase` and `phase_simple` do NOT need to be columns; they are shown here as columns solely to address the query across relational data**
- Any project where `current_status` is `active` should utilize the `current_phase` query for the label
- Could be completed in conjunction with #7635 but the changes in #7635 should be done prior to this work

**Comment by zappfyllc** [comment]
@johnclary @amenity I added the data model above. I believe `current_phase` and `phase_simple` should just be queries resulting from the existing relationships (we can have the assigned dev verify these) while `current_status` should be set via UI actions. Then, we can have the application logic dictate whether to show the `status_name` or the `phase_name` based on @johnclary's suggestions in the parent epic and matching the design by @patrickm02L.

**Comment by zappfyllc** [comment]
@johnclary per grooming today and your point, I'm emphasizing here that `current_status` should still be a simple status (active, cancelled, on hold, archived, etc.), separate from the UI rules that go into showing the badge, and I do think we could refer to this as `status_badge` if we at some point believe it should go into a database view as @frankhereford and @mateoclarke suggested we might handle it in the future.

**Comment by amenity** [comment]
@sergiogcx are these colors accessible by their variable name in MUI? If not, we might should add them to [the Moped theme](https://github.com/cityofaustin/atd-moped/blob/main/moped-editor/src/theme/index.js), right?

**Comment by sergiogcx** [comment]
@amenity @zappfyllc as I am working on this, I am realizing that the majority of projects will not have any phases. Should we hide the chip if the project does not have a phase?

**Comment by amenity** [comment]
@sergiogcx - really good point... @johnclary @zappfyllc - what do you think about implementing logic where a project must have at least one active phase at all times?

**Comment by amenity** [comment]
Circling back on the colors. @patrickm02L thoughtfully chose existing MUI variables for the chips. Can you confirm whether or not we can call those smoothly, @sergiogcx ? Otherwise I'd like to define our own warning/error/info colors in the theme.

**Comment by sergiogcx** [comment]
Just had a brief conversation, and yes those are easy colors to access in MUI. 👍 🌈

**Comment by amenity** [comment]
@patrickm02L I jumped on a call w/ @sergiogcx and confirmed that the variables you specced are working 💯, meaning they're called directly from MUI core. 
![Screen Shot 2021-11-24 at 12 34 10 PM](https://user-images.githubusercontent.com/1463708/143296088-b72897d3-0624-431a-9ee3-91cebb9ff1d5.png)

**Comment by amenity** [comment]
@patrickm02L @sergiogcx -Not a particularly useful contribution 🙈  but while I was obsessing over existential palette questions I updated XD with a ["Status display" artboard](https://xd.adobe.com/view/74e0ba56-e00f-49d8-9a35-7ec2cf144554-21b3/screen/40a77190-463a-49e2-8f72-c6135ad45052/) with the statuses as components.
